lib/tokens/token.js

```diff
@@ -41,41 +41,13 @@ module.exports = function (log, config, random, P, hkdf, Bundle, error) {
     this.createdAt = details.createdAt || 0
```

Comment:
There's this perhaps unintuitive contract in the `Token` API, which is: 

- `Token.create` is for generating brand new tokens, and so `createdAt = Date.now()`
- `Token.from` (`fromHex`, `fromId`) is for building a pre-existing `Token`, and so `createdAt` should definitely be provided
- `new Token` doesn't know which context it is being used in, so just defaults to believing the token is from 1969 until told otherwise.